src/frontend/apps/lti_site/components/VideoWidgetProvider/widgets/DownloadVideo/index.tsx

```diff
@@ -182,11 +188,11 @@ export const DownloadVideo = () => {
           a11yTitle={intl.formatMessage(messages.downloadButtonLabel)}
           color="blue-active"
           download
-          disabled={isResolutionsEmpty}
+          disabled={isResolutionsEmpty || !toggleAllowDownload}
```

Comment:
You can revert this and line 195. The widget is not display when the download is not allowed but in the instructor dashboard we should still be available to download it. Allowing to download the video is only for students.